src/ssl.c

```diff
@@ -3108,22 +3108,32 @@ int wolfSSL_ALPN_GetProtocol(WOLFSSL* ssl, char **protocol_name, word16 *size)
 
 int wolfSSL_ALPN_GetPeerProtocol(WOLFSSL* ssl, char **list, word16 *listSz)
 {
+    int i, len;
+    char *p;
+    byte *s;
+
     if (list == NULL || listSz == NULL)
```

Comment:
Looks like this was a problem before your changes, but we aren't checking that ssl is not NULL here, and we should be.

Reply:
added.